src/boost.mk

```diff
@@ -67,11 +67,11 @@ define $(PKG)_BUILD
     echo 'set(Boost_THREADAPI "win32")' > '$(CMAKE_TOOLCHAIN_DIR)/$(PKG).cmake'
 
     '$(TARGET)-g++' \
-        -W -Wall -Werror -ansi -U__STRICT_ANSI__ -pedantic \
+        -W -Wall -Werror -ansi -std=c++11 -U__STRICT_ANSI__ -pedantic \
         '$(PWD)/src/$(PKG)-test.cpp' -o '$(PREFIX)/$(TARGET)/bin/test-boost.exe' \
```

Comment:
What version of gcc are you using? I had to also add `-DCMAKE_CXX_STANDARD=11` to the cmake test below with gcc5.

Reply:
Oh that makes sense.  I was using gcc7, and I think it is fine with gcc6 as well since from gcc6 on gcc defaults to C++14.  But for gcc5, it would have to be explicitly specified.